packages/session-replay-browser/src/index.ts

```diff
@@ -1,3 +1,4 @@
 import sessionReplay from './session-replay-factory';
 export const { init, setSessionId, getSessionId, getSessionReplayProperties, flush, shutdown } = sessionReplay;
 export { SessionReplayOptions, StoreType } from './typings/session-replay';
+export { SafeLoggerProvider } from './logger';
```

Comment:
I don't see where `SafeLoggerProvider` is used outside of session replay package. We should only export things we want consumers to use.